src/components/Board/ImprovePhraseOutput/ImprovePhraseOutput.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './ImprovePhraseOutput.module.css';
+
+import { Typography } from '@material-ui/core';
+import { PlayArrow } from '@material-ui/icons';
+
+const propTypes = {
+  isRootBoardTourEnabled: PropTypes.bool,
+  isUnlockedTourEnabled: PropTypes.bool,
+  isLocked: PropTypes.bool,
+  disableTour: PropTypes.func.isRequired
```

Comment:
`Warning: Failed prop type: The prop disableTour is marked as required in ImprovePhraseOutput, but its value is undefined.`